{"name": "test_002", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <TestCases.test_01.TestExchange object at 0x03F73910>\n\n    @allure.story('用例2')\n    def test_002(self):\n        _assert = Assert.Assert()\n        print(\"hhhjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj\")\n>       assert _assert.assert_body(1,2)\n\ntest_01.py:64: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Utils.Assert.Assert object at 0x03F61A90>, body = 1, assert_body = 2\n\n    def assert_body(self,body,assert_body):\n        '''\n        请求响应body断言\n        :data body:\n        :data assert_body:\n        :return:\n        '''\n        try:\n>           assert body == assert_body\nE           AssertionError\n\n..\\Utils\\Assert.py:35: AssertionError"}, "attachments": [{"name": "log", "source": "dbc1f7a5-81fc-493d-9fa0-031032c06bb2-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "ea9d7691-458e-4947-a85a-71859ffbb6f0-attachment.txt", "type": "text/plain"}], "start": 1592821802404, "stop": 1592821802405, "uuid": "f1837cd3-a9de-4b0b-8242-038587aa2b37", "historyId": "4852f8a856c7054b2bc974c95a71170a", "fullName": "TestCases.test_01.TestExchange#test_002", "labels": [{"name": "feature", "value": "小额资产兑换"}, {"name": "story", "value": "用例2"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_01"}, {"name": "subSuite", "value": "TestExchange"}, {"name": "host", "value": "DESKTOP-8TLGUH8"}, {"name": "thread", "value": "31572-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_01"}]}